README.md

```diff
@@ -278,12 +278,19 @@ In order to run full test suite, simply execute:
 
 **Run integration tests:**
 
-To run the integration tests, uncomment the following line from `tox.ini` and add the paths to your Kafka and Confluent Schema Registry instances. You can also run the integration tests outside of `tox` by running this command from the source root.
+To run the confluent-kafka-python integration tests you will need access to a running Kafka cluster. In order to execute successfully client will need create, read and write topics. Additionally you will need access to Zookeeper in order to set a request quota for the client `throttled-client`.
```

Comment:
"client `throttled-client`."

What is that referring to?

Reply:
It's referring to the client the example sets the throttle for. I will remove that as part of moving the the example to the test README